e2etest/utils.go

```diff
@@ -76,3 +84,150 @@ func tweakRCPort() func(rc *v1.ReplicationController) {
 	}
 	return nil
 }
+
+// Increment ip and test against CIDR after to avoid overflows.
```

Comment:
I didn't review the code below in detail but it seems like there may be go library functions that could help here?